spec/feature/home_page/root_spec.rb

```diff
@@ -31,4 +31,21 @@
 
     expect(page).to have_button("Register")
   end
+
+  it "does not display a login or register button once a user is logged in" do
+    user = User.create!(username: "eDog", email: "elah@email.com", password: "password")
+    allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user)
+    visit '/login'
+
+    fill_in :email, with: 'eelah@email.com'
+    fill_in :password, with: user.password
```

Comment:
This wouldn't log the user in since the credentials are bad and don't match those of the actual user. Email just needs to be filled in to user.email and this test will be good to go